Draft-Accepted/RFCNNNN-Get-WhatsNew.md

```diff
@@ -0,0 +1,231 @@
+---
+ RFC: RFC00XX
+ Author: Jason Helmick
+ Status: Draft
+ SupercededBy: N/A
+ Version: 1.0
+ Area: Core
+ Comments Due: 4/30/2022
+ Plan to implement: Yes
+---
+# Get-WhatsNew
+
+[Get-WhatsNew](https://github.com/PowerShell/WhatsNew) is a cmdlet that provides information about
+changes and new features for a version of PowerShell, delivered to the local terminal experience.
+
+## Motivation
+
+Customers are unaware of benefits, new features and changes, that could impact their automation,
+performance and security. Today, this useful information is provided through
+[Microsoft Docs](https://docs.microsoft.com/powershell/scripting/whats-new/what-s-new-in-powershell-70)
+and the PowerShell GitHub repository. PowerShell customers would benefit from being able to get this
+information in the terminal as they expect.
+
+- Customers expect to learn about available features that may enable new solutions
+- Customers expect version specific information available from within PowerShell to make upgrade
+  decisions.
+
+> As an admin,
+> I can list the new features released in my current version of PowerShell, similar
+> to [webview](https://docs.microsoft.com/powershell/scripting/whats-new/what-s-new-in-powershell-72)
+> so that I can discover and implement new features in managing my products.
+
+> As a developer,
+> I can list the new features released in my current version of PowerShell, similar
+> to [webview](https://docs.microsoft.com/powershell/scripting/whats-new/what-s-new-in-powershell-72)
+> so that I can discover and implement new features to develop automation solutions.
+
+## Goals/Non-Goals
+
+Goals
+
+- Be distributed as a PowerShell module on the Gallery
+- Support disconnected scenarios (data ships with module)
+- Supports Windows PowerShell 5.1
+- Supports the ability to compare versions
+- Supports a message-of-the-day (motd) option
+
+Non-goals/Alternatives
+
+- Data endpoint (Microservice) to synchronously deliver or update the data.
+- Changelog information
+
+Future feature considerations
+
+- Render markdown output in the console with ANSI styling (enabled by switch parameter)
+- Use ANSI strings to make hyperlinks clickable for terminals that support it
+- Add `Get-Changelog` cmdlet to display Changelog information
+
+## Solution
+
+Ship a **Microsoft.PowerShell.WhatsNew** module on the PowerShell Gallery that contains the cmdlet
+and would:
+
+- Be discoverable using normal PowerShell conventions
+- Be available down-level to Windows PowerShell 5.1
+
+## Specification
+
+This specification is based on the cmdlet [Get-WhatsNew](https://github.com/PowerShell/WhatsNew)
+
+- Module Name: Microsoft.PowerShell.WhatsNew
+- Cmdlet name: Get-WhatsNew
+- Alias: gwn
```

Comment:
We discussed this many times - aliases bring more problems than benefits. 
This must be delegated users. Every user can define aliases in profile which the user prefers in his session.

Reply:
yup, every knowledgeable user can define/re-define aliases in their profile. Should that prevent me from creating a default alias for users that don't wish or have knowledge to do that?

Reply:
I think it's fine that this command gets an alias because I imagine it will primarily be used from the console but he does have a valid point. I have 2178 cmdlets/functions on my system if all those commands had their own alias there would be a lot of conflicts and it would be hard to add new aliases that could easily be remembered.  
There's many built-in aliases that I suspect aren't seeing much usage, either because the commands aren't really useful outside of scripts or because the alias names are too hard to remember. For example, does anyone actually use any of these: `rv` `rp` `stz` `dnsn` `ipal`?

Reply:
@iSazonov @MartinGC94 - I certainly respect both of your view points. I expect this cmdlet to get used, but not on the freq that could normally benefit from an alias.  Thanks for raising this and will consider.